f-engrave.py

```diff
@@ -2223,6 +2223,28 @@ def createWidgets(self):
             self.DoIt()
             if self.cut_type.get() == "v-carve":
                 self.V_Carve_It()
+
+		
+            # A couple quick notes:
+            # - It looks like the command line logic here is just dumping the contents of
+            #   self.gcode (an array of strings) out to the std out (the console)
+            # - The functions below *seem* to be the ones that relate to the GUI click handlers
+            #   and *seem* to add additional strings to the self.gcode array
+            # - I'm not really sure which order these functions should be called in... I'm 
+            #   guessing the calc ones first then the write.  You might need to try re-arranging them
+            # - If you can figure out (through testing) what arrangement is correct for the clean and
+            #   and v-clean options then hooking them up to the command line options should be pretty
+            #   straightforward
+		
+            # use one OR the other of these
+            bit_type = "straight"       # Write_Clean_Click	
+            bit_type = "vbit"           # Write_V_Clean_Click
+            
+            self.V_Carve_It() # Seems to be the important part of VCARVE_Recalculate_Click
+            self.Clean_Path_Calc(bit_type)

```

Comment:
So i guess this error in this Clean_Path_Calc, and there is no statusbar attribute?

if i comment out 2244 it works like normal with no cleanup tool paths being sent to the console

C:\Users\Brando\Desktop\f-engrave CLI changeup>python f-engrave.py -g sq.ngc -f sq.dxf -b
File/Directory Not Found:       /Library/Fonts
(F-Engrave Batch Mode)
Traceback (most recent call last):
  File "f-engrave.py", line 9703, in <module>
    app = Application(root)
  File "f-engrave.py", line 1888, in __init__
    self.createWidgets()
  File "f-engrave.py", line 2244, in createWidgets
    self.Clean_Path_Calc(bit_type)
  File "f-engrave.py", line 8079, in Clean_Path_Calc
    self.statusbar.configure( bg = 'yellow' )
AttributeError: 'Application' object has no attribute 'statusbar'